<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>670</width>
    <height>639</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RPL Sink</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="btnConnect">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnreset">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>70</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnincrement">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>100</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Increment DTSN</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>561</width>
      <height>421</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Debug</string>
     </attribute>
     <widget class="QLineEdit" name="ledstn">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>340</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="ledodaid">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>200</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>290</y>
        <width>46</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Version</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="leinstance">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>250</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="leversion">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>290</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>10</y>
        <width>46</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Received</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>180</y>
        <width>46</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Sent</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>340</y>
        <width>46</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>DSTN</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="tesent">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>220</y>
        <width>311</width>
        <height>141</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="teRcvdata">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>30</y>
        <width>311</width>
        <height>121</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>250</y>
        <width>46</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Instance</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>15</x>
        <y>200</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>dodagid</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Application</string>
     </attribute>
     <widget class="QTextEdit" name="teDatatosend">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>70</y>
        <width>281</width>
        <height>121</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btnsendData">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Send Data</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="teDatarecieved">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>240</y>
        <width>271</width>
        <height>121</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="leAddr">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>90</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>70</y>
        <width>46</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Address</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnPing">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Send Ping</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>50</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Data to send</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>220</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Received Data</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Routing Table</string>
     </attribute>
     <widget class="QTextEdit" name="teRoutingtable">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>60</y>
        <width>351</width>
        <height>261</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLineEdit" name="lePort">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>30</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>COM8</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>40</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Port</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>670</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
